src/TestIds.ts

```diff
@@ -22,6 +22,8 @@ export const TestIds = {
   BANNER: '',
   INTERSTITIAL: '',
   REWARDED: '',
+  GAM_BANNER: '/6499/example/banner',
+  GAM_INTERSTITIAL: '/6499/example/interstitial',
```

Comment:
Looked like the default app open was missing, but most curiously, why not include the test ids for all the GAM types? (https://developers.google.com/ad-manager/mobile-ads-sdk/android/test-ads)

```suggestion
  APP_OPEN: '',
  BANNER: '',
  INTERSTITIAL: '',
  REWARDED: '',
  GAM_APP_OPEN: '/6499/example/app-open',
  GAM_BANNER: '/6499/example/banner',
  GAM_INTERSTITIAL: '/6499/example/interstitial',
  GAM_REWARDED: '/6499/example/rewarded',
```

Also, could you exercise these in the test app?
Just pick class-based or hook-based and cpy-paste the existing ad type test blocks but put in the GAM_ test ids and change the log statements / paths of course to include GAM, to make sure they work?